recipes/ampligone/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage("ampligone", max_pin="x") }}  
```

Comment:
we had in the main branch only the single x ... not sure what is correct x or x.x.x